src/ol/format/TopoJSON.js

```diff
@@ -200,7 +200,9 @@ function concatenateArcs(indices, arcs) {
       // reverse arc
       arc = arcs[~index].slice().reverse();
     }
-    coordinates.push.apply(coordinates, arc);
+    for (vertex of arc) {
+      coordinates.push(vertex);
+    }
```

Comment:
```suggestion
    for (let a = arc.length - 1; a >= 0; --a) {
      coordinates.push(arc[a]);
    }
```

Reply:
Avoiding `reverse()` makes sense but won't you now need separate forward and reverse `for` loops?  Isn't the `slice()` also redundant?

Reply:
Yes, hard to see that in the diff view.